style.css

```diff
@@ -0,0 +1,151 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.navbar {
+  background-color: #3c3a39;
+  padding: 20px;
+}
+
+.navbar-wrapper {
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.navbar-logo {
+  text-decoration: none;
+  font-size: 20px;
+  color: white;
+  font-family: "Inter", sans-serif;
+}
+
+.nav-links {
+  list-style-type: none;
+}
+
+.nav-link {
+  display: none;
+}
+
+.menu {
+  width: 30px;
+  filter: invert(100%);
+}
+
+.headline-section {
+  background-color: #131110;
+  width: 100%;
+  height: 600px;
+  display: flex;
+  position: relative;
+  overflow: hidden;
+  justify-content: center;
+  align-items: center;
+}
+
+.headline-section-wrapper {
+  padding: 10px;
+}
+
+.headline-title {
+  color: #f55800;
+  font-size: 50px;
+  font-family: "Crete Round", serif;
+  font-weight: 400;
+}
+
+.headline-professional {
+  color: white;
+  font-family: "Roboto", sans-serif;
+  font-weight: 700;
+  font-size: 32px;
+  margin-bottom: 10px;
+}
+
+.headline-description {
+  color: #f4f5f7;
+  font-family: "Poppins", sans-serif;
+  font-weight: 400;
+  line-height: 32px;
+  font-size: 20px;
+}
```

Comment:
- [ ] The correct `font-size` for this element is `16px` and not `20px`. Kindly use the correct `font-size` so that your design looks identical to that of the Figma template that you chose. 👍 😎

###  Your Design

![image](https://user-images.githubusercontent.com/77106962/234968333-c184e230-32a5-4dca-9ec6-d57fed9cc98c.png)

### Figma Template

![image](https://user-images.githubusercontent.com/77106962/234968376-9b8529a7-0adf-41f1-a415-f1dc4944ce2a.png)